x-pack/plugins/observability/public/pages/alerts/containers/alerts_page/alerts_page.tsx

```diff
@@ -142,7 +142,7 @@ function AlertsPage() {
   }, []);
 
   const manageRulesHref = config.unsafe.rules.enabled
```

Comment:
@mgiota, I'm thinking if we need to update the feature flag to align it with this update. e.g. `xpack.observability.unsafe.alerts.rules.enabled` ?

Reply:
Good thought! Let me merge this PR to hide the menu option from sidebar for now and we could do it in another PR. I am hoping that we will soon get rid of this flag completely, so I wouldn't worry much about it now.